index.html

```diff
@@ -1,262 +1,209 @@
-                <li>CANOPY</li>
-                <li>Back End Dev</li>
-                <li>2015</li>
-                
-            </ul>
-            <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
-        <ul class="tech_list">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
+    <section class="content">
+        <h1 class="intro">I’m Shahnawaz Ali Glad to see you!</h1>
+        <p class="intro1">
+            I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h3 class="intro2">LET’S CONNECT</h3>
+        <ul class="social">
+            <li class="social-icon twitter">
+                <a><img src="images/twitter.png" alt="twitter"></a>
+            </li>
+            <li class="social-icon linkedin">
+                <a><img src="images/Linkedin.png" alt="linkedin"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="images/m.png" alt="m"></a>
+            </li>
+            <li class="social-icon github">
+                <a><img src="images/github.png" alt="github"></a>
+            </li>
+            <li class="social-icon fingers">
+                <a><img src="images/fingers.png" alt="fingers"></a>
+            </li>
         </ul>
-        <button class="btn1" type="submit">See Project</button>
-    </div>
-    
-    </div>
-    
-    <div class="work_card">
-    <img class="mobi-3" src="./images/thirdimage.png" alt="card" id="img_third"/>
-    <img class="desk-1" src="./images/deskthird.png" alt="card" />
-    <div class="side_info">
-    <h3 class="project_name">
-            Multi-Post
-            Stories
-        </h3>
-    
-        <ul class="first_list">
-            <li>CANOPY</li>
-            <li>Back End Dev</li>
-            <li>2015</li>
-            
-        </ul>
-        <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
-    <ul class="tech_list">
-        <li>html</li>
-        <li>css</li>
-        <li>javaScript</li>
-    </ul>
-    <button class="btn1" type="submit">See Project</button>
-    </div>
-    </div>
-    <div class="work_card">
-           
-        <img class="mobi-4" src="./images/fourthimage.png" alt="card"/>
-        <img class="desk-1" src="./images/deskfourth.png" alt="card" />
-        <div class="side_info">
-        <h3 class="project_name">
-            Multi-Post
-            Stories
-        </h3>
-    
-        <ul class="first_list">
-            <li>CANOPY</li>
-            <li>Back End Dev</li>
-            <li>2015</li>
+    </section>
+    <section id="portfolio" class="project-section">
```

Comment:
You have a horizontal scroll in the mobile version that is most likely being brought about by this section. 

- [ ] Ensure that this project section has the same width compared with the sections above and below it.